base/exports.jl

```diff
@@ -893,6 +893,11 @@ export
     chown,
     cp,
     ctime,
+    DiskStats,
+    disk_available,
```

Comment:
Perhaps my suggestion wasn't so clear, but my idea was to only introduce one new name, `diskstat`, and if you want available space etc you do
```
dstat = diskstat(path)
avail = dstat.available
used = dstat.used
total = dstat.total
```

Reply:
Ah yeah that makes way more sense. Thanks!